src/modules/page/room/RoomDetailsSwitch.page.vue

```diff
@@ -39,10 +37,7 @@ const { deactivateRoom, fetchRoom, resetState } = roomDetailsStore;
 const { canCreateRoom } = useRoomAuthorization();
 
 const canAccessRoom = computed(() => {
-	return (
-		(envConfigModule.getEnv.FEATURE_ROOMS_ENABLED && canCreateRoom.value) ||
-		envConfigModule.getEnv.FEATURE_ROOM_ADD_STUDENTS_ENABLED
-	);
+	return canCreateRoom.value;
```

Comment:
```suggestion
const { canCreateRoom: canAccessRoom } = useRoomAuthorization();
```

Because of `canCreateRoom` equals `canAccessRoom`, I think using it like this is better.